web-bundle/src/main/java/com/graphhopper/replica/StreetEdgeExporter.java

```diff
@@ -40,6 +41,7 @@ public class StreetEdgeExporter {
     public static final CSVFormat CSV_FORMAT = CSVFormat.DEFAULT.withHeader(COLUMN_HEADERS);
 
     private Map<Long, Map<String, String>> osmIdToWayTags;
+    private Map<Long, Map<Boolean, String>> osmIdToDirectionToAccessFlags;
```

Comment:
i think it'd be a little safer to store a `Set<String>` accessFlags instead of a single string. Hashsets don't provide guarantees on their iteration or toString ordering, so as written it's possible we'd error on two gh edges in the same way with the same access flags in a different order. doesn't seem likely and unit tests are passing, but i found some [horror stories (even with identical insertion order!)](https://stackoverflow.com/questions/2704597/iteration-order-of-hashset) so would prefer to avoid this possibility altogether 🙂

Reply:
Ooo good catch, I totally missed the possibility of this being an issue! That wouldn't been a fun one to debug... 🤕 

Fixed in [78f6ea9](https://github.com/replicahq/graphhopper/pull/171/commits/78f6ea90b5eb73d885853da9d8b281467105bb48)